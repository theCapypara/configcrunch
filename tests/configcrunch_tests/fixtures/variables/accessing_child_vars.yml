base:
  str_field: "{{ level_direct.name }}"
  level_direct:
    name: "{{ more.name }}"
    more:
      name: Name of Level
  more:
    name: Name of Base
    copy: "{{ level_direct.name }}"