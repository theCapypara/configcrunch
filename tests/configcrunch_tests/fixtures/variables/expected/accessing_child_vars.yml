base:
  str_field: Name of Level
  level_direct:
    name: Name of Level
    more:
      name: Name of Level
  more:
    name: Name of Base
    copy: Name of Level